src/openzaak/components/zaken/models/zaken.py

```diff
@@ -881,6 +911,37 @@ def unique_representation(self):
         )
         return f"({self.zaak.unique_representation()}) - {betrokkene.rsplit('/')[-1]}"
 
+    def _get_relation_for_betrokkene_type(self):
+        """
+        Return the details of the related betrokkene.
+
+        Depending on the betrokkeneType, return the related object holding the
+        ``betrokkeneIdentificatie`` details. This relation may not be set if a
+        betrokkene URL is specified.
+
+        TODO: check amount of DB queries on repeated calls.
+        """
+        match self.betrokkene_type:
+            case RolTypes.natuurlijk_persoon:
+                return self.natuurlijkpersoon
+            case RolTypes.niet_natuurlijk_persoon:
+                return self.nietnatuurlijkpersoon
+            case RolTypes.vestiging:
+                return self.vestiging
+            case RolTypes.organisatorische_eenheid:
+                return self.organisatorischeeenheid
+            case RolTypes.medewerker:
+                return self.medewerker
+            case _:
+                raise ValueError("Unknown rol betrokkene type")
+
+    @property
+    def betrokkene_identificatie(self):
+        """
+        Expose ``betrokkene_identificatie`` accessor to API serializers.
+        """
+        return self._get_relation_for_betrokkene_type()
```

Comment:
helper method is not necessary, implementation can move directly inside the property (and then we can also cache the result?)